src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/channel.yml

```diff
@@ -6,7 +6,7 @@ sylius_admin_api_channel:
         identifier: code
         alias: sylius.channel
         section: admin_api
-        only: ['show', 'index']
+        except: ['update']
```

Comment:
Is there any reason not to include `update`?

Reply:
Not sure, I thought it is like good practice as in country.yml, locale.yml, currency.yml in the same directory

Reply:
Mentioned examples do not contain almost any data. The channel should be editable